backend/Dockerfile

```diff
@@ -1,25 +1,30 @@
 # Use an official lightweight Python image as the base
-FROM python:3.11
+# 3.11 required by mir
+FROM python:3.11-slim
 
-# Set the working directory in the container
-WORKDIR /app
-
-# Copy the forecastbox directory into the container
-COPY forecastbox /app/forecastbox
-COPY pyproject.toml /app/pyproject.toml
+# Install system deps
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt update && \
+    apt install -y --no-install-recommends git \
```

Comment:
why do we need git? Is it because of those pip installs later? I have opened 3 PRs today to get rid of that, ie, everything (except forecastbox) should be pypi-available like today or tomorrow